contribs/gmf/src/sass/vars.scss

```diff
@@ -3,3 +3,37 @@
 @import "~bootstrap/scss/_functions.scss";
 @import "~bootstrap/scss/_variables.scss";
 @import "~@fortawesome/fontawesome-free/scss/_variables.scss";
+
+
+$table-border-color: $border-color !default;
+
+:root {
+  --brand-primary: #{$brand-primary};
```

Comment:
missing at least:
`--danger`
`--back-color`

(also in vars_only file)

Reply:
Not in vars only because they are defined by bootstrap, thanks :-)